### ISSUE TITLE
Yuzu GUI doesn't start up, unless the driver is updated/reinstalled and Windows isn't rebooted

### ISSUE THREAD

**Issue by HRudyPlayZ** [yuzu-emu/yuzu#12324]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1645

### Description of Issue

- When trying to open the emulator, the main window doesn't open.
- The emulator's process will still run in the background and take ressources
- This doesn't happen when the graphics driver is hotswapped by an update/reinstall, but reappears again after a Windows restart.
- Sometimes, the emulator will have 2 processes running in the background

### Expected Behavior

The GUI should launch properly.

### Reproduction Steps

- Attempt to launch the emulator.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/13625658/yuzu_log.txt)

### System Configuration

CPU: Ryzen 7 3700x
GPU/Driver: Nvidia RTX 3070 (driver 546.29)
RAM: 32 GB
OS: Windows 10 Pro

**Comment by Squall-Leonhart** [comment]
Not a yuzu bug, consult a general pc forum for help.

**Comment by HRudyPlayZ** [comment]
It only affects Yuzu though, Satisfactory and many other Vulkan games don't have any issue.

**Comment by Squall-Leonhart** [comment]
Yuzu isn't even using Vulkan at the time it starts, it is QPaint using a custom software raster, the reason i said to consult a general pc forum is because your issue will be connected to Startup or scheduled tasks which shoves themselves into every process, I'm not going to go into that here in the bugs comments because its out of scope, but Asus GPU Tweak II in the past has caused this very issue, so start with anything that can potentially draw into client windows.

**Comment by HRudyPlayZ** [comment]
But wouldn't this cause the window to stay visible in the taskbar but nothing appearing?
I mean i can try turning off every starting app, but i don't see this being caused by any of them. Give me a few minutes.

It's also not to my knowledge a Qt issue since i managed to run Qt5, and Qt6 apps not so long ago, admittedly i don't think i've tried running QtQuick stuff but it should work fine too.

**Comment by HRudyPlayZ** [comment]
Ok found the culprit, apparently Sunshine on Windows breaks the Yuzu GUI, it makes sense since it's almost the only software i installed since the bug occurs. I'll leave this here in case this is actually a Yuzu bug and will make an issue on Sunshine's side as well.

**Comment by HRudyPlayZ** [comment]
Does Yuzu use DXGI capture in some way? The sunshine dev mentioned that it could be a case of the 2 apps fighting each other over this particular feature.

**Comment by Squall-Leonhart** [comment]
No, but Sunshine uses Vigem and possibly controller hiding functionality that Yuzu can time out on while reading in the availabl controllers.
